e2e/pod.go

```diff
@@ -314,6 +314,9 @@ func waitForPodInRunningState(name, ns string, c kubernetes.Interface, t int, ex
 					return true, err
 				}
 			}
+		case v1.PodUnknown:
+			e2elog.Logf("State of the pod cannot be obtained")
+			return false, nil
```

Comment:
```suggestion
		case v1.PodUnknown:
			e2elog.Logf(
			"%s app  is in %s phase expected to be in Running  state (%d seconds elapsed)",
			name,
			pod.Status.Phase,
			int(time.Since(start).Seconds()))
			return false, nil
```
Since this is the only other possible state, move log at line 321 to here.